### DISCUSSION TITLE
Baseplate cornering. Can we have them removed (maybe optionally)?

### DISCUSSION THREAD

**Discussion by ExplodingFist** [Le0Michine/FusionGridfinityGenerator#100] [Ideas]
Right now the cornering on the base-plates is less than ideal, as they don't connect as well together (little gap where they both corner out), but more importantly, it's difficult to extrude padding between the base plate and the wall of a drawer because you can only easily extrude the flat area before the corner, so it looks off when printed and connected. Ideally we could get outer edges that are square to make things easier to work with. Thoughts?

Love this project BTW and use it to generate stuff daily right now.

**Comment by Le0Michine** [comment]
it seems like there are two things mentioned here which might be different

- being able to add padding to ensure the base doesn't slide around the drawer. That's something I'm hoping to add to the main functionality so that you won't have to deal with it manually
- corner fillets, the main reason they are there is to improve printability, many printers out where don't have linear advance/pressure advance dialed in which leads to bumps on sharp corners among other things which could prevent proper baseplates alignment, so this is meant to workaround the issue. Also I just like rounded corners as it makes the look a little be more finished (I can see that being subjective though)

if you weren't need to add padding to the base manually, would you care about corners being round or is there some other reason as well? I'm a bit hesitant regarding adding settings for individual features like that as the UI is already quite crowded and I really want to keep it useful while being simple at the same time.

PS as a work around for now it should be relatively simple to select fillets you don't need and press delete to get sharp corners back. Let me know if that doesn't work for some reason

**Reply by Le0Michine** [reply]
in progress 
<img width="543" alt="image" src="https://github.com/Le0Michine/FusionGridfinityGenerator/assets/2707008/66eb7944-0419-4362-a7f6-2981e63a8dac">
